deployments/outputs.json

```diff
@@ -0,0 +1,78 @@
+      "ConstantAuctionPriceCurve": "0x75834f8929EcE6b7a0D09E7200FDE14E5b4C1F65",
+      "InitialCollateralSet": "0xC8655Dd4AE34f61Ce2eA28E7069707fdc8E1fdfb",
+      "BitEthRebalanceManager": "0xad0E4a2bD56Cf01bB3D4A68600700c2cd955Af81",
+      "BitEthRebalancingSetToken": "0x38aE0ac4bf28eb9A7EbC45f78C5400E4db718E73"
+    },
+    "state": {
+      "last_deployment_stage": 1,
+      "network_id": 42
+    }
+  },
+  "development": {
+    "addresses": {
+      "ERC20Wrapper": "0x2C530e4Ecc573F11bd72CF5Fdf580d134d25f15F",
+      "EIP712Library": "0x72D5A2213bfE46dF9FbDa08E22f536aC6Ca8907e",
+      "OrderLibrary": "0x2eBb94Cc79D7D0F1195300aAf191d118F53292a8",
+      "ExchangeIssueLibrary": "0x5315e44798395d4a952530d131249fE00f554565",
+      "RebalancingHelperLibrary": "0xDFF540fE764855D3175DcfAe9d91AE8aEE5C6D6F",
+      "StandardProposeLibrary": "0xC1bE2c0bb387aa13d5019a9c518E8BC93cb53360",
+      "StandardSettleRebalanceLibrary": "0xda54ecF5A234D6CD85Ce93A955860834aCA75878",
+      "StandardStartRebalanceLibrary": "0x33DeF1aA867Be09809F3a01CE41d5eC1888846c9",
+      "StandardPlaceBidLibrary": "0x10A736A7b223f1FE1050264249d1aBb975741E75",
+      "StandardFailAuctionLibrary": "0xb125995F5a4766C451cD8C34C4F5CAC89b724571",
+      "WBTC": "0xc7124963Ab16C33E5bF421D4c0090116622B3074",
+      "WETH": "0xC6B0D3C45A6b5092808196cB00dF5C357d55E1d5",
+      "Vault": "0x7209185959D7227FB77274e1e88151D7C4c368D3",
+      "TransferProxy": "0x3f16cA81691dAB9184cb4606C361D73c4FD2510a",
+      "Core": "0x99356167eDba8FBdC36959E3F5D0C43d1BA9c6DB",
+      "SetTokenFactory": "0x45B3A72221E571017C0f0ec42189E11d149D0ACE",
+      "WhiteList": "0xdD66C23e07b4D6925b6089b5Fe6fc9E62941aFE8",
+      "RebalancingSetTokenFactory": "0x4609e0ED27A8BAAc57b753D36a5D2971915588f9",
+      "SignatureValidator": "0x4ef5b1E3dA5573466Fb1724D2Fca95290119B664",
+      "ExchangeIssueModule": "0xd7e3593d3d8A22480e2136EaB9497286D87C0231",
+      "IssuanceOrderModule": "0x2579D2B186BbA16999016dB077b4874A7520f92e",
+      "RebalanceAuctionModule": "0x404C55a936f3006B13B020efAaf5771A600Ec04d",
+      "RebalancingTokenIssuanceModule": "0xfD946D47d3dB1e06126d16281Fb3E222A1bA8179",
+      "PayableExchangeIssue": "0x96EccEa4E124322a6aA0a004da1b91d9a3024C73",
+      "TakerWalletWrapper": "0xDDb2B738682AD218eD87CF6f3a466798644e5d8D",
+      "KyberNetworkWrapper": "0xd2aa8d362b1CaA68553642831b86Abb3D24B4579",
+      "ZeroExExchangeWrapper": "0x434f1EB003B78c0EAbe034313F1aFf47920e0860",
+      "LinearAuctionPriceCurve": "0x46c6A737C75cE3a58c6b2De14970E8841c72DcEF",
+      "ConstantAuctionPriceCurve": "0x1a488d7B42C1Ec1539b78f772BF13eCCB723f5fa",
+      "BitEthRebalanceManager": "0x57b7aB09008dD5eda262b160A7b0f0d17c8754B4",
+      "InitialCollateralSet": "0x22d34df714907847839addd687E2fd725d25d3a9",
+      "BitEthRebalancingSetToken": "0x0d751881E89BAA7e2238e9F9CEE54606EAf716b0"
+    },
+    "state": {
+      "last_deployment_stage": "5"
```

Comment:
So do the deployments restart using the previous state AS well as taking into account already-deployed contracts?

Reply:
Yup.